src/Kupo/Data/Http/Error.hs

```diff
@@ -31,6 +31,15 @@ invalidPattern =
                  \<https://cardanosolutions.github.io/kupo>!"
         }
 
+invalidOutputPattern :: Response
+invalidOutputPattern =
+    responseJson status400 Default.headers $ HttpError
+        { hint = "Invalid output reference pattern! To fetch matches, you may provide any valid \
+                 \output reference. Make sure to double-check the documentation at: \
```

Comment:
Suggestion: indicates in the error the shape of the pattern to help users. Something along the lines of:

> you may provide any valid output reference in the form of {transaction-id}/{output-index}. The transaction id must be base16-encoded and 64 characters long.